source/includes/openshift/deploy-minio-on-openshift.rst

```diff
@@ -70,39 +71,38 @@ Select the tab that corresponds to your preferred installation method:
 
 .. tab-set::
 
-   .. tab-item:: Red Hat Marketplace
+   .. tab-item:: Red Hat OperatorHub
```

Comment:
Switched order of these tabs, because using OperatorHub saves you from several annoyances and we should encourage it. 

Marketplace does lead you to OperatorHub, but in a roundabout way that few people do. (I.e. paying Red Hat instead of us directly.)

Reply:
"changes" from here through line 105 are all because of the tab reorder. (Improvements to existing welcome, as always.)